catalyst-toolbox/src/rewards/veterans.rs

```diff
@@ -77,13 +77,35 @@ fn calc_final_eligible_rankings(
         .collect()
 }
 
+fn calc_final_ranking_consensus_per_review(rankings: &[impl Borrow<VeteranRankingRow>]) -> Decimal {
```

Comment:
let's not duplicate code, this can be done together with `calc_final_ranking_per_review` as a sort of score confidence

Reply:
OK, I'm going to try, good Rust exercise

Reply:
ok done in 6d47932